src/wasm-binary.h

```diff
@@ -390,8 +390,8 @@ class WasmBinaryWriter : public WasmVisitor<WasmBinaryWriter, void> {
   void writeMemory() {
     if (wasm->memory.max == 0) return;
     if (debug) std::cerr << "== writeMemory" << std::endl;
-    o << int8_t(BinaryConsts::Memory) << int8_t(log2(wasm->memory.initial))
-                                      << int8_t(log2(wasm->memory.max))
+    o << int8_t(BinaryConsts::Memory) << int8_t(ceil(log2(wasm->memory.initial)))
+                                      << int8_t(ceil(log2(wasm->memory.max)))
```

Comment:
I thought the number had to be a power of 2? I guess it makes sense to have `ceil` if it doesn't have to be.